src/Pages/OfficerDB/InfoRow.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { Modal, Button, ModalHeader, ModalBody } from 'reactstrap'
+import Editor from './Edits/Editor'
+
+const api = require('../../APIFunctions/OfficerDB')
+const svg = require('./SVG')
+
+export default function Row (props) {
```

Comment:
Row is already a reactstrap component
```suggestion
export default function InfoRow (props) {
```